README.md

```diff
@@ -75,6 +77,8 @@ NOTE: Make sure you can run either <code>python</code> or
 (5) Open your favorite browser and run <code>localhost:3000</code> or
 <code>127.0.0.1:3000</code>.
 
+NOTE: Always use **ctrl + c** command to quit the **HTTPServer** to prevent **socket.error:[Errno 48]**.
```

Comment:
How did you trigger the socket error?

Reply:
If we directly close the application's tab in the web browser or we directly close the terminal in which the server is running and then if i again try to run server using ```npm run serve``` i get this socket error.